doc/design/validation.md

```diff
@@ -0,0 +1,25 @@
+# Requirements for being a valid DANDI asset
+
+## General
+1. All assets must have a non-zero file size.
+
+## NWB files
+- subject id
+- age in iso8601 interval format or datetime of birth
+- age reference 
+- sex
+
+### NWB organization disambiguation requirements
+
+At least one of the following should be present. in case of having to disambiguate
+- session
+- sample
```

Comment:
I am not sure if there is a generic notion of a "sample" in NWB, so I guess it is for the `tissue_sample_id` [*].

[*] forgot that it is coming probably from `ndx-dandi-icephys` and wasted time to search it up in the main nwb schema, uff

Reply:
and what about `slice_id`?  might have both I guess (multiple slices from the same sample).

I guess in our metadata schema they both (tissue and slice) would be `BioSample` but of different  `sampleType`, right?

Reply:
correct.

Reply:
was that is the answer to the last question, or to both? ;-)

Reply:
the last question (biosample of different sampletype) as implemented currently in the extractor.